app/policies/application_policy.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    false
+  end
+
+  def create?
+    false
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    false
+  end
+
+  def edit?
+    update?
+  end
+
+  def destroy?
+    false
+  end
```

Comment:
why are these methods required?

Reply:
These methods are defined in ApplicationPolicy to provide default behavior for authorization in Pundit. They provide template for common actions so we dont need to rewrite these methods again and again in evry policy.